trino/trino.go

```diff
@@ -739,6 +780,32 @@ func (st *driverStmt) exec(ctx context.Context, args []driver.NamedValue) (*stmt
 	if err != nil {
 		return nil, fmt.Errorf("trino: %v", err)
 	}
+
+	// initial progress callback call
```

Comment:
Why do we need special code to handle initial call?

Reply:
Because it takes small amount of time before rows.fetch is executed, some short queries might end too fast with first callback with `FINISHED` state even if callback period is set fast. This seems to be weird as all intermediate steps is skipped, so I added initial call to update as soon as we submitted the query.

Reply:
fair. 

Yet please move the comment to be above 
```
srStats := QueryProgressInfo{
			QueryId:    sr.ID,
			QueryStats: sr.Stats,
		}
```

currenntly it is mispositioned.